torchao/quantization/quant_api.py

```diff
@@ -199,6 +201,22 @@ def change_linear_weights_to_int4_woqtensors(
 ########
 # TO BE DEPRECATED END
 ########
+def dequantize_float8_training(model: nn.Module) -> nn.Module:
```

Comment:
we want to inline this function in `_replace_with_custom_fn_if_matches_filter` for now?

Reply:
Done